tsconfig.storybook.json

```diff
@@ -3,7 +3,8 @@
     "esModuleInterop": true,
     "jsx": "react",
     "skipLibCheck": true,
-    "preserveConstEnums": true
+    "preserveConstEnums": true,
+    "resolveJsonModule": true
```

Comment:
forgot to ask, can't we declare `json` as a module in order for our TS to recognize it instead of adding TS rule ?

Reply:
It would also be possible to declare it in the type definition but I didn't see any gain in this. Could you elaborate on the benefits of your suggestion?

Reply:
it is OK, with that actually, but the reason I mentioned that is that we can define modules in their own definition TS file and avoid adding multiple rules which makes things a bit confusing sometimes due to lots of rules(for me as example), and I had some kind of issue with this one using another lib before(e.g angular), but since TS v2.9 we can use what you did out there, so, all fine with that :)